Volha_Ivanova/Module_4_Framework/Design_Patterns/my-app/src/main/java/com/epam/cdp/volha_ivanova/task1/fileHandlerFactory/FileManager.java

```diff
@@ -0,0 +1,13 @@
+package com.epam.cdp.volha_ivanova.task1.fileHandlerFactory;
+
+public class FileManager {
+	public FileHandler getFileHandler(String type) {
+		
+		if ("TXT".equals(type)) {
```

Comment:
1. It'a better to throw exception in case of unsupported type of FileHandler
2. "TXT", "XML" should be the Enum FileHandleType. Your method should take Enum as parameter.